WordPressSecurityWhitePaper.html

```diff
@@ -87,13 +87,22 @@ <h4>A2 - Broken Authentication and Session Management</h4>
 <h4>A3 - Cross Site Scripting (XSS)</h4>
 <p>WordPress provides a range of functions which can help ensure that user-supplied data is safe<sup id="ref10"><a href="#footnote10">10</a></sup>. Trusted users, that is administrators and editors on a single WordPress installation, and network administrators only in WordPress Multisite, can post unfiltered HTML or JavaScript as they need to, such as inside a post or page. Untrusted users and user-submitted content is filtered by default to remove dangerous entities, using the KSES library through the <code>wp_kses</code> function.</p>
 
-<p>As an example, the WordPress core team noticed before the release of WordPress 2.3 that the function <code>the_search_query()</code> was being misused by most theme authors, who were not escaping the function's output for use in HTML. In a very rare case of slightly breaking backward compatibility, the function's output was changed in WordPress 2.3 to be pre-escaped.</p>
+<p>WordPress' range of security-related functions include:</p>
+
+<ul>
```

Comment:
Is there anywhere in the developer hub we could link to explaining these in depth? Or do we not want folks to leave the document? Perhaps as a footnote?